Sprint-1/fix/median.js

```diff
@@ -3,8 +3,19 @@
 // If you're in the Sprint-1 directory, you can run `npm test -- fix` to run the tests in the fix directory
 
 function calculateMedian(list) {
+  let median;
```

Comment:
The arrays specified in `median.test.js` happened to be sorted.

Normally, before we can easily find the median value in an array, we need to sort the array first.
Can you try inserting a statement to sort the numbers in `list` first?